crates/jstzd/tests/octez_client_test.rs

```diff
@@ -61,6 +61,55 @@ async fn generates_keys() {
     assert_eq!(secret_keys["name"], alias);
 }
 
+#[tokio::test]
+async fn show_address() {
+    let temp_dir = TempDir::new().unwrap();
```

Comment:
Not to be addressed in this PR: now quite a few test cases have these `dir` lines. Perhaps we can refactor a bit. It's fine for now.